ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/radiobutton/v1/radiobutton/radiobutton.html

```diff
@@ -43,7 +43,7 @@
               <label class="cmp-adaptiveform-radiobutton__option-label"
                     title="${radioButton.tooltipVisible ? '' : radioButton.tooltipText}">
                   <input type="radio"
-                        name="${radioButton.name}"
+                        name="${radioButton.id}"
```

Comment:
Test cases are failing due to this change, shouldn't name be the radio button's name ? Can you also add repeatable test case in the existing repeatable test suite ?

Reply:
No. Different radio button in the same group should have same names but not across the different groups. 
Radio Button Groups can have same name across panels/repeating panels, hence we should use id since id is unique for different groups